store/root.js

```diff
@@ -47,6 +47,13 @@ class Root {
     }
   }
 
+  // eslint-disable-next-line class-methods-use-this
+  async logout() {
+    // TODO: /logout should ideally be moved to /internal
+    await apiRequest('https://api.dev.core.ac.uk/logout')
+    window.location.replace('/logout.html')
+  }
```

Comment:
Do you add this method just because to do not like having link to API in the UI?

Reply:
I don't mind having API in UI. But in that case, logout has to work like this `https://api.dev.core.ac.uk/logout?continue=someurl`. 

Also, it's nice to have some action in the store. For example, when we store something in local storage this is the right place for removing it from there.

Reply:
I agree. On the other hand, having another action that can be avoided is not that good. Less code is less errors what it better.